README.md

```diff
@@ -97,3 +97,8 @@ This repo also defines the production data for funds and rounds in JSON format i
 The fields that can be overridden are: opens, deadline, assessment_deadline. So to override the opening date:
 
         export force_opens_R2W3=2023-01-01
```

Comment:
how is this picked up in the round?

Reply:
It's here in `round_data`: https://github.com/communitiesuk/funding-service-design-fund-store/blob/32feb7b79f063d9da243dd1169783b22e92e2fb3/core/data_operations/round_data.py#L48

Reply:
okay great thanks @srh-sloan